src/profile/components/profileInformation/profileInformationAccountManagementLinkUtils.ts

```diff
@@ -52,3 +52,7 @@ export function getAmrUrl(authenticationMethodReference: AMRStatic): string {
       );
   }
 }
+
+export function hasTunnistusSuomiFiAmr(profile: Profile | null): boolean {
+  return profile?.amr === tunnistusSuomifiAMR;
```

Comment:
According to [the OIDC specification](https://openid.net/specs/openid-connect-core-1_0.html#IDToken), the `amr` claim is an **array of strings**. It could be that Tunnistamo erroneously provides a single string. I think both styles should be handled here, a single string, and an array of strings.

Reply:
Very good point, I didn't know Tunnistamo is not compliant with the specs. I checked that Helsinki Tunnistus (Keycloak) returns an array in amr. But Profiili does not use Tunnistus (yet).

Tunnistamo never returns an array, so the possibility should not be added to Profiili at the moment. The code would be redundant. It is clear this must be changed when Tunnistamo won't be used anymore.

Reply:
That would be knowingly leaving a bug in the code waiting to bite. I wouldn't leave it there. Who remembers to fix it on time before it bites?

Reply:
From Profile's point of view the bug is in Tunnistamo. :)

I'll change this to be future proof.

Maybe report a bug for Tunnistamo?

Reply:
> ...the bug is in Tunnistamo. :)
> Maybe report a bug for Tunnistamo?

Yep, that's known. Unfortunately it's difficult to change because it would break backward compatibility :disappointed:

Reply:
Tests added and amr is now converted to "string[]".